### PR TITLE
[Docs] SIEM 7.4 network map

### PR DESCRIPTION
**Pull request by benskelker**
Documentation for the SIEM network map.

[Preview](http://stack-docs_513.docs-preview.app.elstc.co/guide/en/siem/guide/master/siem-ui-overview.html)

### CODE REVIEW

File: docs/en/siem/siem-ui.asciidoc [hunk 1]
```diff
@@ -65,29 +65,164 @@ versions, machine type, and so forth.
...
-investigation time enrichment, and provides network event tables that enable
-interaction with the Timeline. You can drag and drop items of interest from the
-Network view to Timeline for further investigation.
+The Network view provides key network activity metrics and an interactive map,
+facilitates investigation time enrichment, and provides network event tables that
+enable interaction with the Timeline. You can drag and drop items of interest from
+the Network view to Timeline for further investigation.
 
 [role="screenshot"]
 image::network-ui.png[]
```
Comment (spong) [inline docs/en/siem/siem-ui.asciidoc]: May need one more iteration on this screenshot as the maps styling has been updated from red/green to blue/magenta for source and destination points.

Comment (benskelker) [reply docs/en/siem/siem-ui.asciidoc]: Thanks - updated the screenshot

**Comment by benskelker** [comment]
@karenzone and @tsg - do you have any comments/corrections or can I merge this?
Thanks,

File: docs/en/siem/siem-ui.asciidoc [hunk 2]
```diff
@@ -65,29 +65,164 @@ versions, machine type, and so forth.
...
+----------------------------------
+<1> The value of this field must be the same as the ingest pipeline name in
+<<geo_pipeleine, step 1>> (`geoip-info` in this example).
+
+[float]
+[[map-actions]]
+==== Interact with the map
+Hover over source and destination points to see more information, such as hostnames
+and IP addresses. To drill down, click a point and use the plus sign to add a field
+to the query bar. You can also click a hostname to jump to the SIEM Host page, or
```
Comment (spong) [inline docs/en/siem/siem-ui.asciidoc]: Sorry for the last-minute comment, but just thought of this -- you can only click on the hostname & source.ip/destination.ip if you set up a [URL field formatter](https://www.elastic.co/guide/en/kibana/current/field-formatters-string.html) for these fields in the relevant indices.

Here's the necessary config for each:

Field: `host.name`
URL Template: `[BASE_URL]/app/siem#/network/ip/{{value}}`
Label Template: `{{value}}`




Field: `source.ip` / `destination.ip`
URL Template: `[BASE_URL]/app/siem#/network/ip/{{value}}`
Label Template: `{{value}}`

Comment (benskelker) [reply docs/en/siem/siem-ui.asciidoc]: No problem. I added the steps, but I'm debating putting all map configuration steps on a separate page. What do you think?
I'll also consult with @karenzone

Comment (benskelker) [reply docs/en/siem/siem-ui.asciidoc]: @spong please don't review yet - I'm going to put the configuration instructions on a separate page. I'll ping you when it's ready - probably tomorrow.

Comment (spong) [reply docs/en/siem/siem-ui.asciidoc]: Sounds good! I think that's the right move. Then that keeps the main guide page specific to each overview item without going into the weeds of configuration.

Comment (benskelker) [reply docs/en/siem/siem-ui.asciidoc]: @spong 
Updated with map configuration on its own page. Thanks,

Comment (spong) [reply docs/en/siem/siem-ui.asciidoc]: Much better flow with the configuration on its own page -- very nice! Thanks @benskelker! 🙂

**Pull request merged**